map_model/src/objects/road.rs

```diff
@@ -407,6 +407,17 @@ impl Road {
         result
     }
 
+    // lane must belong to this road. Offset 0 is the centermost lane on each side of a road, then
```

Comment:
I used `offset` before and was so confused why it wasn't counting from the center. :(  So I restored this "deprecated" method. I still need to figure out how to name "offset" from center vs left side of the road better.